mypy/test/testpep561.py

```diff
@@ -188,7 +188,9 @@ def test_get_pkg_dirs(self) -> None:
         dirs = get_site_packages_dirs(sys.executable)
         assert dirs
 
-    @pytest.mark.skipif(hasattr(sys, 'base_prefix') and sys.base_prefix != sys.prefix, reason="Temporarily skip to avoid having a virtualenv within a venv.")
+    @pytest.mark.skipif(sys.platform == 'darwin' and hasattr(sys, 'base_prefix') and \
+    sys.base_prefix != sys.prefix, \
+    reason="Temporarily skip to avoid having a virtualenv within a venv.")
```

Comment:
You don't need the backslashes, because it's inside `(...)`. Also, please reindent the continuation lines to line up (i.e. `sys` and `reason` in the 2nd and 3rd lines should start in the same column as `sys` on the previous line).

Reply:
Fixed, cheers!